src/diffusers/pipelines/deprecated/stable_diffusion_k_diffusion/__init__.py

```diff
@@ -25,7 +25,7 @@
     ):
         raise OptionalDependencyNotAvailable()
 except OptionalDependencyNotAvailable:
-    from ...utils import dummy_torch_and_transformers_and_k_diffusion_objects  # noqa F403
+    from ....utils import dummy_torch_and_transformers_and_k_diffusion_objects  # noqa F403
 
     _dummy_objects.update(get_objects_from_module(dummy_torch_and_transformers_and_k_diffusion_objects))
 else:
```

Comment:
Not sure whether to deprecate K Diffusion. The approach is still reasonably popular, it's just that the original implementation isn't very actively maintained, so thought it made sense to move ours to deprecated as well. I'm cool moving it back if we feel strongly against deprecating.

Reply:
I think it's ok! it's not that they cannot use it anymore